frontend/src/components/Menu.jsx

```diff
@@ -38,5 +38,10 @@ export const Menu = [
     title: 'Register',
     path: '/Register',
     cName: 'dropdown-link'
+},
+{
```

Comment:
Comment component should not appear in the dropdown menu. Please remove this component.